src/components/Contacts/GiftStatus/GiftStatus.tsx

```diff
@@ -12,6 +12,20 @@ export enum GiftStatusEnum {
   Late,
   Hidden,
 }
+export const GiftIsLateStatus = (
```

Comment:
What do you think of calling this in the `GiftStatus` component instead of exporting it and making the parent call it? You'd have to change the GiftStatusProps to take the date instead of the status enum... I think it'd be cleaner and keep the implementation isolated in this component.